File: angular/src/Recipe.ts
export class Recipe{
    constructor(public RecipeName:string, public RecipeCategory:string, 
        public RecipePreperingTime:number, public RecipeDifficultLevel:number,
         public RecipeAddDate:Date,public RecipeIngredients:string[],
          public RecipePreparation:string[], public RecipeuserCode:string,
          public RecipePicture:string, public RecipeIf:boolean){}
}


    

File: WebApplication2/Models/Recipe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication2.Models
{
    public class Recipe
    {
        public string RecipeName { get; set; }
        public string RecipeCategory { get; set; }
        public int RecipePreperingTime { get; set; }
        public int RecipeDifficultLevel { get; set; }
        public DateTime RecipeAddDate { get; set; }
        public List<string> RecipeIngredients { get; set; }
        public List<string> RecipePreparation { get; set; }
        public string RecipeuserCode { get; set; }
        public string RecipePicture { get; set; }
        public bool RecipeIf { get; set; }
    }
}
        
          
        